tools/Voron.Recovery/CommandLineApp.cs

```diff
@@ -67,16 +75,49 @@ private static void ConfigureRecoveryCommand()
                 var ignoreInvalidJournalErrorsArg = cmd.Option("--IgnoreInvalidJournalErrors", "Default is false.", CommandOptionType.SingleValue);
                 var ignoreDataIntegrityErrorsOfAlreadySyncedTransactionsArg = cmd.Option("--IgnoreInvalidDataErrorsOfAlreadySyncedTransactions", "Default is false.", CommandOptionType.SingleValue);
                 var ignoreInvalidPagesInARowArg = cmd.Option("--IgnoreInvalidPagesInARow", "Default is false.", CommandOptionType.SingleValue);
-
                 var loggingModeArg = cmd.Option("--LoggingMode", "Logging mode: Operations or Information.", CommandOptionType.SingleValue);
-
                 var masterKey = cmd.Option("--MasterKey", "Encryption key: base64 string of the encryption master key", CommandOptionType.SingleValue);
+                var recoveredServerPath = cmd.Option("--RecoveredServerPath", "Recovered database server destination", CommandOptionType.SingleValue);
 
                 cmd.OnExecute(() =>
                 {
+                    if (recoveredServerPath.HasValue() == false)
```

Comment:
I don't want to create a server here. Instead, you should be creating an instance of `DocumentDatabase()` and use that API directly.

Reply:
Please elaborate, how can I create DocumentDatabase instance without ServerStore? (and therefor a RavenServer) without refactoring DocumentDatabase?